crates/ruff_cli/src/args.rs

```diff
@@ -107,6 +107,9 @@ pub struct CheckArgs {
     /// Output serialization format for violations.
     #[arg(long, value_enum, env = "RUFF_FORMAT")]
     pub format: Option<SerializationFormat>,
+    /// Specify file to write output to (default: stdout).
```

Comment:
This should make clear that the output means lint output, not e.g. the fixed code output

Reply:
Can you expand on "fixed code outout"? Are you referring to the updated files on disk?

Here, output comprises of everything that goes to stdout which includes the diagnostics, `--show-*` output, `--diff` output, statistics.

Reply:
Yep, someone could assume `ruff --fix my_file.py -o my_file_fixed.py` would work